### DISCUSSION TITLE
toBest({system:??}) does not exist?

### DISCUSSION THREAD

**Discussion by oscarthroedsson** [convert-units/convert-units#342] [General]
Hello!
Really nice tool and seems like a fun project. I am trying to use re-factor a lot in my nutrition webb-app and trying to use:
```
convert()
          .from("g")
          .toBest({ system: "imperial" }),
```
But the type look like this:
```js
 toBest(
            options?: { exclude?: Unit[] | undefined; cutOffNumber?: number | undefined },
        ): { val: number; unit: string; singular: string; plural: string };
```

The code I am trying to use is in your readME file, and you may take that away. I was wondering if you have something similar to, toBest when it comes to metric to imperial?

**Comment by Taar** [comment]
That feature was added in the [v3.0.0-beta.3](https://github.com/convert-units/convert-units/releases/tag/v3.0.0-beta.3) so make sure you're using a beta version equal or greater than that. The `README.md` file when you visit the repository is always reflective of the state of the code in the main branch. Make sure you select the tag for the version you're using otherwise you'll won't be looking at the correct documentation.

**Reply by oscarthroedsson** [reply]
Thanks for clarification 🙏🏼
